lib/wallet/src/Cardano/Wallet.hs

```diff
@@ -1633,6 +1639,43 @@ balanceTransaction ctx change pp ti wallet ptx = do
             otherErr
                 -> throwE otherErr
 
+increaseZeroAdaOutputs
+    :: forall era. WriteTx.RecentEra era
+    -> WriteTx.PParams (Cardano.ShelleyLedgerEra era)
+    -> Cardano.Tx era
+    -> Cardano.Tx era
+increaseZeroAdaOutputs era pp (Cardano.Tx cardanoBody keyWits) =
```

Comment:
Would like to move along with `balanceTransaction` to some `Write.Tx.Balance` module*. I think it's better to merge this first, but I probably should get to it near-immediately afterwards.

*) Ultimately `increaseZeroAdaOutputs` should preferably be decoupled from `balanceTransaction`, but for the medium-term I think this should be fine.